### DISCUSSION TITLE
export model to  Torch Script

### DISCUSSION THREAD

**Discussion by fouvy** [skorch-dev/skorch#959] [Q&A] [answered]
How to export my model to torch script? For example, transformers export model to torch script: https://huggingface.co/docs/transformers/torchscript

**Comment by BenjaminBossan** [comment, marked as answer]
In general, the PyTorch `Module` is available as an attribute called `module_` on the skorch net. So e.g.:

```python
net = NeuralNet(MyModule, ...)  # MyModule is your custom or pretrained pt module
net.fit(X, y)
module = net.module_  # this is the trained MyModule instance
traced_module = torch.jit.trace(module, ...)
```

It is not possible to trace the `net` itself, only the PyTorch `Module`.

I haven't worked with torch script, so if the provided solution not work, please report back.
